scripts/serve/serve.js

```diff
@@ -11,6 +11,10 @@ const webpackServe = (options) => {
     config,
     ...(port) && { port },
     ...(host) && { host },
+    dev: { stats: {
+      colors: true,
+      children: false,
```

Comment:
Curious, does this turn the logs off?

Reply:
no, just the children one that spams us. This for example:
```[./node_modules/css-loader/lib/css-base.js] 2.21 KiB {mini-css-extract-plugin} [built]
Child mini-css-extract-plugin node_modules/css-loader/index.js??ref--5-1!node_modules/postcss-loader/lib/index.js??postcss!node_modules/sass-loader/lib/loader.js!node_modules/terra-navigation-side-menu/lib/NavigationSideMenu.scss:
    Entrypoint mini-css-extract-plugin = *```